week05/TodoAdd.js

```diff
@@ -0,0 +1,21 @@
+function TodoAdd({todoList,setTodoList}){
+    function addTodo(event){
+        event.preventDefault();
+        const newTodoItem={
+            id: Date.now(),
+            text: event.target["todo-text"].value,
+            done:false,
+        }
+        setTodoList([...todoList,newTodoItem]);
+        event.target.reset();
+    }
+    return(
+        <form className="todo-add" onSubmit={addTodo}>
+            <input id="input1" type="text" name="todo-text" required placeholder="오늘의 할 일은?"></input>
+            <input id="input2" type="submit" value="ADD"></input>

```

Comment:
id 명칭은 단순한 텍스트보다는 의도가 명확히 드러나도록 짓는 것이 좋습니다.
이는 팀 프로젝트에서 다른 개발자와 코드를 공유할 때 원할한 소통을 위해 중요합니다.

```suggestion
            <input id="text-input" type="text" name="todo-text" required placeholder="오늘의 할 일은?"></input>
            <input id="add-btn" type="submit" value="ADD"></input>
```